### ISSUE TITLE
[Enhancement]: avoir un paramètre pour désactiver les vibrations.

### ISSUE THREAD

**Issue by Supertolek** [PapillonApp/papillon-v7#339]
### Description du problème

C'est assez dérangeant des fois d'avoir son téléphone qui vibre, c'est assez inhabituel.

### Description de l'amélioration

Avoir un toggle pour désactiver les vibrations.

**Comment by Gabriel29306** [comment]
Note pour les dev:
Utiliser les flags pour simplifier le travail

**Comment by Supertolek** [comment]
Je pense que la meilleure solution serait de créer une nouvelle catégorie dans personnalisation avec tous les paramètres identiques à celui-ci, si jamais d'autres suggestions de ce genre sont rapportées.
Mais c'est pas vraiment à moi de prendre cette décision, étant donné que ça modifie quand même l'interface.
Si quelqu'un qui passe par là souhaite donner son avis, il est le bienvenue.

**Comment by Kgeek33** [comment]
ton idée est intéressante @Supertolek 
en plus, on pourrait ajouter des options tel que la désactivation des sons ou tout simplement désactiver certains éléments dans chaque page (un exemple facile à trouver : le graphique d'évolution de la moyenne générale)

**Comment by Supertolek** [comment]
Oui! Bon, faudra attendre qu'un dev se libère. (je préviens tout de suite, je risque de reporter tout les bugs que je vois,mais j'essaierai d'être synthétique et de pas surcharger tout le monde)

**Comment by Kgeek33** [comment]
ma pr #325 fixe tous les bugs connus
profite-en, elle n'a pas encore été merge, j'essaierai de corriger les bugs que tu rencontres le plus vite possible

**Comment by Supertolek** [comment]
Ok.
Ça, c'est plutôt une amélioration, mais le bouton home reset l'animation lorsqu'on appuie plusieurs fois dessus de suite. Ce serait trop stylé que ça le fasse pas.
Ensuite, lorsqu'on utilise le datetimepicker pour sélectionner le jour pour lequel on veut connaitre nos devoirs, la page charge mal, et devient vide. En plus, le sélecteur de date bug et demande 2 fois la date.
Voilà pour ce que j'ai pour l'instant.

**Comment by Kgeek33** [comment]
Alors ton amélioration, je vais vérifier ça
Sinon les 2 derniers bugs ont été corrigé sur ma pr #325 :)

**Comment by Gabriel29306** [comment]
Le mieux serait une page accessibilité qui: 
- Activer/Désactiver les vibrations
- Activer/Désactiver les sons
-  Permettre une app en échelle de gris (comme Pronote) (fonctionnalité la plus compliquée)

Et peut-être d'autres

**Comment by Kgeek33** [comment]
Echelle de gris ? Est-ce que ça vaut le coup ?
Je suis pas contre mais une intégration longue pour que peu de personnes active cette fonctionnalité

**Comment by Gabriel29306** [comment]
> Echelle de gris ? Est-ce que ça vaut le coup ?
> Je suis pas contre mais une intégration longue pour que peu de personnes active cette fonctionnalité

C'est un extra, c'est vraiment pas obligatoire

**Comment by godetremy** [comment]
> Le mieux serait une page accessibilité qui: 
> - Activer/Désactiver les vibrations
> - Activer/Désactiver les sons
> -  Permettre une app en échelle de gris (comme Pronote) (fonctionnalité la plus compliquée)
> 
> Et peut-être d'autres

Ton magnifique téléphone a déjà tout ces réglages dans le panneau d'accessibilité, je vois pas pourquoi s'embêter ! Surtout le niveau gris, a la limite les deux autres je comprends !

**Comment by Gabriel29306** [comment]
> > Le mieux serait une page accessibilité qui: 
> > ...
> 
> Ton magnifique téléphone a déjà tout ces réglages dans le panneau d'accessibilité, je vois pas pourquoi s'embêter ! Surtout le niveau gris, a la limite les deux autres je comprends !

Oui, pour le nouveau de gris, c'est parce que Pronote l'a. Pour les autres c'est un peu comme les jeux, je veux bien avoir le son ici mais pas là, pareil pour les vibrations

**Comment by codeuriii** [comment]
est ce que la page qui enlève les vibrations a déja été faite ?
